docs/resources/Channel.md

```diff
@@ -153,13 +159,47 @@ Represents a message sent in a channel within Discord.
 | height | integer | height of thumbnail |
 | width | integer | width of thumbnail |
 
+###### Embed Image Structure
+
+| Field | Type | Description |
+|-------|------|-------------|
+| url | string | source url of image |
```

Comment:
do you think these should have a special type "url" - as it must be a url that starts with https?://